fleet/lib/opensearch-logging/overlays/cp/ingress.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    nginx.ingress.kubernetes.io/proxy-ssl-verify: "false"
+    cert-manager.io/cluster-issuer: letsencrypt-dev
```

Comment:
`letsencrypt-dev` only exists on ayekan (and I just realize this is missing from `cert-manager-conf`).  Please use `letsenscrypt` as the clusterissuer.

Reply:
Handled in [a351ffe](https://github.com/lsst-it/k8s-cookbook/pull/231/commits/a351ffef68dae4174a2c962909c187c64c7e0b5a)